index.html

```diff
@@ -0,0 +1,396 @@
+                                          consequat.
+                                    </p>
+                                    <p>
+                                        Duis aute irure dolor in reprehenderit in voluptate velit esse
+                                        cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+                                        proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                                    </p>
+                                    <p>
+                                        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+                                        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+                                        quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+                                        consequat.
+                                    </p>
+                                    <p>
+                                        Please check in regularly for updates and we expect that you register at our site.
+                                    </p>
+                                    <p>
+                                        Best Regards,
+                                    </p>
+                                    <p>
+                                        <div class="">
+                                            Dennis Antony
+                                        </div>
+                                        <div class="">
+                                            Founder
+                                        </div>
+                                    </p>
+
+                    </div>
+                </section>
+            </div>
+            <div class="section s4">
+                <section id="content_business_logic">
+                    <div class="about_heading">
+                        <h2>
+                            Business Logic
+                        </h2>
+                    </div>
+                    <p>
+                        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+                    </p>
+                    <p>
+                        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam
+                    </p>
+                    <p>
+                        quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+                    </p>
+                    <p>
+                        consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+                    </p>
+                    <p>
+                        cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+                    </p>
+                    <p>
+                        proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                    </p>
```

Comment:
Don't use separate p tags for each line of words. Use a single p tag for a block of text. Use <br> or adjust max width for desired look and feel.